app/views/moderator_applications/_form.html.slim

```diff
@@ -0,0 +1,55 @@
+section.card
+  header.card-header
+    strong Moderator Application Form
+
+  .card-body
+    - if application.errors.any?
```

Comment:
It just occurred to me that I copy and pasted this code from the submissions form. It should probably be factored out into another partial view instead.

Reply:
Call should look like:
```ruby
render_validation_errors(model: application, model_noun: 'application')
```
or, like I said elsewhere, even better that information should be dynamically retrievable from the object itself, so that you don't have to pass it, so just:
```ruby
render_validation_errors application
```

Reply:
Yeah, that sounds fair, but let's leave that refactor in the future.